scripts/apps/search/views/search-parameters.html

```diff
@@ -110,24 +110,25 @@
             </div>
         </div>
 
-        <div class="multiple">
-            <div class="field" ng-if="repo.archive && search_config.creator">
-                <label for="search-creator">
-                    {{:: 'Creator' | translate}}
-                </label>
-                <select id="search-creator"
-                        ng-model="fields.original_creator"
-                        ng-options="user._id  as user.display_name for user in userList track by user._id">
-                <option value="" label=""></option>
-                </select>
-            </div>
+        <div class="field" ng-if="repo.archive && search_config.creator">
+            <label for="search-creator">
+                {{:: 'Creator' | translate}}
+            </label>
+            <select id="search-creator"
+                    ng-model="fields.original_creator"
+                    ng-options="user._id  as user.display_name for user in userList track by user._id">
+            <option value="" label=""></option>
+            </select>
+        </div>
 
-            <div class="field" ng-if="search_config.spike">
-                <label for="search-spike">
-                {{:: 'In Spiked' | translate}}
-                </label>
-                <span sd-switch ng-model="fields.spike"></span>
-            </div>
+        <div class="field" ng-if="search_config.spike">
+            <label for="spike-options">
+                {{:: 'Spiked Content' | translate}}
+            </label>
+            <select id="spike-options" name="spike-options" ng-model="fields.spike">
+                <option value="include" label="{{:: 'Include spiked content' | translate }}"></option>
```

Comment:
@akintolga  Once you select `Include spiked content` or `Spiked only content`. You cannot exclude spiked content i.e. the default search without spiked content. You have to clear filters. Can you include the option i.e. exclude spiked content?